lib/email.rb

```diff
@@ -76,4 +76,22 @@ def self.obfuscate_part(part)
       part[0] + "*" * (part.size - 2) + part[-1]
     end
   end
+
```

Comment:
I don't follow that, why do we need to settings for?

Reply:
Ah I guess I don't need that for now, will add that in a later PR, it is for autofilling the settings in the UI